tooling/gradle/integration_tests.gradle

```diff
@@ -0,0 +1,13 @@
+sourceSets {
+    integrationTest
+}
+
+// define custom test task for running integration tests
+// This is used by okbuck in order to determine vm args and env variables
+task integrationTest(type: Test) {
+    description = 'Runs the integration tests.'
+    group = 'verification'
+    testClassesDirs = sourceSets.integrationTest.output.classesDirs
+    classpath = sourceSets.integrationTest.runtimeClasspath
+    outputs.upToDateWhen { false }
+}
```

Comment:
since this can be reused by modules, we should move it into `.okbuck/gradle` generated folder so users can then include it directly rather than having to define one.

Reply:
On second thought I think it will bring some cyclic dependency issues. if the file is not there okbuck could fail if a project is applying this file. 

we can leave as is and ask users to create a file and apply from there